### PR TITLE
Ruby rewrite adsense v2

### PR DESCRIPTION
**Pull request by donovanfm**

### CODE REVIEW

File: v2/ruby/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+This is a set of simple samples written in Ruby, which provide a minimal
+example of Google AdSense integration within a command line application.
+
+This starter project provides a great place to start your experimentation into
+the AdSense Management API.
+
+## Prerequisites
+
+Please make sure that you're running Ruby 2.5+ and you've run
+`gem install` in this directory to install all prerequisites.
```
Comment (abadreldin) [inline v2/ruby/README.md:12]:
```suggestion
`bundle install` in this directory to install all prerequisites.
```

Comment (abadreldin) [reply v2/ruby/README.md:12]: Or alternatively `gem install google-apis-adsense_v2`

File: v2/ruby/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+## Setup Authentication
```
Comment (abadreldin) [inline v2/ruby/README.md:14]: Suggestion to add a comment on populating the client_secrets file.

File: v2/ruby/README.md (hunk 1 above)
Comment (donovanfm) [reply v2/ruby/README.md]: Going with the second option.

**Review by abadreldin — APPROVED** [review]
LGTM

**Pull request merged**